packages/redux/src/promotionEvaluations/actions/fetchPromotionEvaluationItems.ts

```diff
@@ -0,0 +1,16 @@
+import { fetchPromotionEvaluationItemsFactory } from './factories';
+import { getPromotionEvaluationItems } from '@farfetch/blackout-client/promotionEvaluations';
+
+/**
+ * Fetches a brand with the given id.
```

Comment:
I believe this description doesn't belongs here 😂

Reply:
Effects of copy/paste 😄  Fixed 🙂